cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1069,7 +1077,9 @@ public MapAndStatus(T loadMap, K status) {
                         return Future.succeededFuture();
                     } else if (kafka.getSpec().getCruiseControl() == null) {
                         LOGGER.warnCr(reconciliation, "Kafka resource lacks 'cruiseControl' declaration : No deployed Cruise Control for doing a rebalance.");
-                        return updateStatus(reconciliation, kafkaRebalance, new KafkaRebalanceStatus(),
+                        KafkaRebalanceStatus status = new KafkaRebalanceStatus();
+                        status.addCondition(StatusUtils.buildWarningCondition("Cruise Control disabled", "Kafka resource lacks 'cruiseControl' declaration"));
```

Comment:
Again, this condition string `Cruise Control disabled` needs to moved to a variable/enum referenced here and in the `onNew` method. Again the condition, exception and log warning message should probably be the same.